go/libkb/util_windows.go

```diff
@@ -184,3 +188,47 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	}
 	return nil
 }
+
+// Notify the shell that the thing located at path has changed
+func notifyShell(path string) {
+	shChangeNotifyProc.Call(
+		uintptr(0x00002000), // SHCNE_UPDATEITEM
+		uintptr(0x0005),     // SHCNF_PATHW
+		uintptr(unsafe.Pointer(syscall.StringToUTF16Ptr(path))),
+		0)
```

Comment:
One concern about unsafe pointer code like this: Are we racing against the garbage collector? `StringToUTF16Ptr` is going to return a newly allocated buffer, but after we convert it to a `Pointer` I don't think there are any references to it remaining. What happens if the GC decides to clean it up while `shChangeNotifyProc` is still running?